### PR TITLE
Functional Tests : Migration TS (BO > Advanced Parameters 04 => 07)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : Migration TS (BO > Advanced Parameters 04 => 07)
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| Related PRs       | N/A
| How to test?      | Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/3600093246
| Possible impacts? | N/A

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/01_downloadSampleFiles.ts [hunk 1]
```diff
@@ -1,30 +1,25 @@
...
 
 // Import pages
-const dashboardPage = require('@pages/BO/dashboard');
-const importPage = require('@pages/BO/advancedParameters/import');
+import dashboardPage from '@pages/BO/dashboard';
+import importPage from '@pages/BO/advancedParameters/import';
 
-const baseContext = 'functional_BO_advancedParameters_import_downloadSampleFiles';
+import {expect} from 'chai';
+import {BrowserContext, Page} from 'playwright';
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/01_downloadSampleFiles.ts]:
```suggestion
import type {BrowserContext, Page} from 'playwright';
```

File: tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/02_importFile.ts [hunk 2]
```diff
@@ -1,38 +1,23 @@
...
+import categoriesPage from '@pages/BO/catalog/categories';
 
-// Import Data
-const {Data} = require('@data/import/categories');
+// Import data
+import {Data} from '@data/import/categories';
 
-const baseContext = 'functional_BO_advancedParameters_import_importFile';
+import {expect} from 'chai';
+import {BrowserContext, Page} from 'playwright';
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/02_importFile.ts]: same here

File: tests/UI/campaigns/functional/BO/14_advancedParameters/06_team/employees/05_sortAndPagination.ts [hunk 3]
```diff
@@ -37,6 +26,12 @@ Pagination
 Delete created employees
  */
 describe('BO - Advanced Parameters - Team : Sort and pagination employees', async () => {
+  const employeeData: EmployeeFaker = new EmployeeFaker();
+
+  let browserContext: BrowserContext;
+  let page: PageTransitionEvent;
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/06_team/employees/05_sortAndPagination.ts]: What's this type ?

**Review by boubkerbribri — COMMENTED** [review]
little comment to fix that are present on some other tests too

File: tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/01_downloadSampleFiles.ts (hunk 1 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/01_downloadSampleFiles.ts]: Fixed

File: tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/02_importFile.ts (hunk 2 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/14_advancedParameters/05_import/02_importFile.ts]: Fixed

File: tests/UI/campaigns/functional/BO/14_advancedParameters/06_team/employees/05_sortAndPagination.ts (hunk 3 above)
Comment (Progi1984) [reply tests/UI/campaigns/functional/BO/14_advancedParameters/06_team/employees/05_sortAndPagination.ts]: Fixed

**Comment by Progi1984** [comment]
@boubkerbribri It's fixed.

**Review by NeOMakinG — APPROVED** [review]
lgtm

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @Progi1984

**Pull request merged**